scripts/eachdist.py

```diff
@@ -669,13 +681,16 @@ def release_args(args):
     cfg.read(str(find_projectroot() / "eachdist.ini"))
     versions = args.versions
     updated_versions = []
+
+    excluded = cfg["exclude_release"]["packages"].split()
```

Comment:
`exclude_release` group from eachdist.ini will now be excluded from this process as they are released independently by their dedicated maintainers.